src/abstract.jl

```diff
@@ -6,18 +6,59 @@ export CutPruner, AbstractCutPruner, addcuts!, ncuts
 
 abstract AbstractCutPruningAlgo
 
+"""
+A cut pruner maintains a matrix `A` and a vector `b` such that
+represents `size(A, 1)` (` == length(b)`) cuts.
+Let `a_i` be `A[i,:]` and `β_i` be `b[i]`, the meaning of the cut depends on the sense.
+Cuts (A, b) defines the half-space satisfying:
+Ax >= b if islb
+Ax <= b otherwise
+If `sense` is
+* `:Min`, then the cut pruner represents the concave polyhedral function `min ⟨a_i, x⟩ + β_i`;
+* `:Max`, then the cut pruner represents the convex polyhedral function `max ⟨a_i, x⟩ + β_i`;
+* `:≤`, then the cut pruner represents the polyhedra defined by the intersection of the half-space `⟨a_i, x⟩ ≤ β_i`;
+* `:≥`, then the cut pruner represents the polyhedra defined by the intersection of the half-space `⟨a_i, x⟩ ≥ β_i`.
+
+Internally, instead of `sense`, the booleans `isfun` and `islb` are stored.
+The mapping between `sense` and these two booleans is given by the following table
+
+| `sense` | `isfun` | `islb` |
+| ------- | ------- | ------ |
+| Min     | true    | false  |
+| Max     | true    | true   |
+| ≤       | false   | false  |
+| ≥       | false   | true   |
+
+"""
 abstract AbstractCutPruner{N, T}
 
+function sense2isfunislb(sense::Symbol)
```

Comment:
I think this name is not that clear. Maybe could we define two functions: 
`getsense(a::Bool, b::Bool)`
`gettype(sense)` 
... if we assume that the tuple of Boolean defines the type of the representation.

What do you think?